website/src/pages/api/inference_auth/[...parts].ts

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+import type { NextApiRequest, NextApiResponse } from "next";
+import { InferenceTokenResponse } from "src/types/Chat";
+
+export default async function inferenceAuthCallback(req: NextApiRequest, res: NextApiResponse) {
+  const { code, parts } = req.query;
+  console.log(req.query);
+  if (!Array.isArray(parts) || parts.length !== 1) {
+    return res.status(400).end();
+  }
+  const [provider] = parts as string[];
+  const url = process.env.INFERENCE_SERVER_HOST + `/auth/callback/${provider}?code=${code}`;
+  const { data } = await axios<InferenceTokenResponse>(url);
+  console.log(data);
```

Comment:
```suggestion
```

Reply:
This function is not currently used anywhere, so I left the console log for debugging purposes, I will remove it in the next PR when I integrate it into the API requests.